app/assets/javascripts/select_doctype.js

```diff
@@ -0,0 +1,55 @@
+// Controls the display of the Document type when creating new assets
+
+Blacklight.onLoad(function() {
+
+var docTypeJson = "/sample_doctypes.json";
+var totalDocTypes;
+//loading json and creating doctype list
+$.getJSON(docTypeJson).done(function(data) {
+	var $docTypeList = $("#docTypes");
+	var docVals = [];
+	$("#asset_type_select").change(function() {
+		var $dropdown = $(this);	
+		$docTypeList.html("");	
+		$( "#subTypes" ).remove();	
+		$docTypeList.append("<option>Please Select</option><option value=''></option>");
+		var key = $dropdown.val();
+		if (key == 'http://definitions.artic.edu/ontology/1.0/type/StillImage') {
+			totalDocTypes = data.asset_types.StillImage.doctypes;
+			}
+		if (key == 'http://definitions.artic.edu/ontology/1.0/type/Text') {
+			totalDocTypes = data.asset_types.Text.doctypes;
+			}
+	
+		for (var i = 0; i < totalDocTypes.length; i++) {
+			docVals = totalDocTypes[i];
+			$docTypeList.append("<option value=" + docVals['value'] + " label=" + docVals['label'] + ">" + docVals['label'] + "</option>");
+		}
+	})
+	
+		
+	//creating subtype list
+	$("#docTypes").change(function() {
+		var $dropdown = $(this);
+		var totalSubtypes = [];
+		$( "#subTypes" ).remove();		
+		var key = $dropdown.val();
+		console.log('key', key);
+		var subTypeVals = [];	
+		if (key) {
+			$("#docTypes").after(" <select class='select required form-control form-control' id='subTypes'><option value=''></option></select>");
```

Comment:
Because we haven't defined a `subTypes` property, this won't do anything at the moment. Once @scossu has identified the additional predicates, we can change this. For now, though, this is fine. It's just that only the first-level document type will actually get set on the asset.